table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -48,22 +49,21 @@
   private final UnderDatabase mUdb;
   private final Map<String, String> mConfig;
 
-  /**
-   * Creates an instance of a database.
-   *
-   * @param context the catalog context
-   * @param type the database type
-   * @param name the database name
-   * @param udb the udb
-   */
+  private DatabaseInfo mDatabaseInfo;
+
   private Database(CatalogContext context, String type, String name, UnderDatabase udb,
-      Map<String, String> configMap) {
+      Map<String, String> configMap, @Nullable DatabaseInfo dbInfo) {
     mContext = context;
     mType = type;
     mName = name;
     mTables = new ConcurrentHashMap<>();
     mUdb = udb;
     mConfig = configMap;
+    if (dbInfo == null) {
+      mDatabaseInfo = new DatabaseInfo();
+    } else {
+      mDatabaseInfo = dbInfo;
```

Comment:
Is there ever a scenario when the `dbInfo` is already populated? Can't we just keep it `null` until it is updated? I think a failed db sync will remove the db. Then, we don't have to pass in `null` as a parameter.

Reply:
done